ppb/flags.py

```diff
@@ -0,0 +1,19 @@
+"""
+ppb.flags contains singletons to communicate various things.
+
+Use like None:
+    * compare against using `is`
+    * Don't instantiate a new instance.
+"""
+
+
+class Flag:
+    """Inherit from Flag to make a simple class flag."""
+    def __new__(cls, *args, **kwargs):
+        return cls
```

Comment:
What happens if you actually instantiate one? (I keep thinking something odd happens with `__init__()` if you return an existing object from `__new__()`.)

Would it be simpler/easier to just raise a `TypeError`?

Reply:
```
>>> flag = Flag(1, 2, 3)
>>> flag is Flag
True
```

We could also raise a TypeError. This is definitely not an API I'm set on quite yet.